src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -171,60 +172,68 @@ docker run <OTHER-DOCKER_OPTIONS> -e CHE_HOST=<ip-addr-or-dns> eclipse/che:<vers
+* `CHE_API_EXTERNAL`: Points to the external API endpoint that is used by the browser clients. Its value is taken from the Che server `CHE_API` variable. Its default value is defined in the `che.properties` variable.
 
-----
-# Browser --> Che Server
-#    1. Default is '${CHE_HOST}:${SERVER_PORT}/wsmaster/api'. In fact, requests are sent to whatever IP/hostname is in your browser address bar
-#    2. Else use the value of che.api
-#
-# Che Server --> Docker Daemon Progression:
-#    1. Use the value of che.infra.docker.daemon_url
-#    2. Else, use the value of DOCKER_HOST system variable
-#    3. Else, use Unix socket over unix:///var/run/docker.sock
-#    4. Else default docker0 IP - 172.17.42.1
-#
-# Che Server --> Workspace Connection:
-#        1. Use the value of che.docker.ip
-#        2. Else, use address of docker0 bridge network, if available
-#
-# Browser --> Workspace Connection:
-#    1. Use the value of che.docker.ip.external
-#    2. Else, use che.docker.ip value
-#    3. Else use value provided by ws container inspect
-#
-# Workspace Agent --> Che Server
-#    1. If set, use value of CHE_INFRA_DOCKER_MASTER__API__ENDPOINT
-#    2. Default is 'http://che-host:${SERVER_PORT}/api', where 'che-host' is IP of docker0 (linux) VM IP (Mac and Win).
-----
+* `CHE_API`: Points to the same value as `CHE_API_INTERNAL` for backward compatibility.
+
+[NOTE]
+====
+The `CHE_API` variable will be removed in a future release.
+====
+
+When it is booting or starting a workspace, Che goes through a progression algorithm to establish the protocol, IP address, and port to establish communication. You can override certain parameters in Che’s configuration to overcome issues with the Docker daemon, workspaces, or browsers being on different networks.
+
+*Browser -> Che server*
+
+The '${CHE_HOST}:${SERVER_PORT}/wsmaster/api' variable is the default. However, requests are sent to the IP/hostname in your browser's address bar. You can also use the value of `che.api`.
+
+*Che server -> Docker daemon progression*
+
+Use the value of the `che.infra.docker.daemon_url` variable. You can also use one of the following values:
+
+* The value of the DOCKER_HOST system variable
 
-It is common for configuration with firewalls, routers, networks and hosts to make the default values we detect to establish these connections incorrect. You can run `docker run <DOCKER_OPTIONS> eclipse/che info --network` to run a test that makes connections between simulated components to reflect the networking setup of Che as it is configured. You do not need all connections to pass for Che to be properly configured. For example, on a Windows machine, this output may exist, just indicating that `localhost` is not an acceptable domain for communications, but the IP address `10.0.75.2` is.
+* The Unix socket over unix:///var/run/docker.sock
+
+* The default docker0 IP - 172.17.42.1
```

Comment:
Likewise, "docker0" seems like it should be styled to me. Elsewhere, IP addresses are also monospaced.